src/main/java/net/sf/jabref/importer/fileformat/MedlineImporter.java

```diff
@@ -84,50 +121,371 @@ public boolean isRecognizedFormat(BufferedReader reader) throws IOException {
+                ArticleTitle articleTitle = (ArticleTitle) object;
+                fields.put("title", removeSurroundingBrackets(articleTitle.getContent().toString()));
+            } else if (object instanceof Pagination) {
+                Pagination pagination = (Pagination) object;
+                String startPage = "";
+                String endPage = "";
+                for (JAXBElement<String> element : pagination.getContent()) {
+                    if ("MedlinePgn".equals(element.getName().getLocalPart())) {
+                        putIfNotNull(fields, "pages", fixPageRange(element.getValue()));
+                    } else if ("StartPage".equals(element.getName().getLocalPart())) {
+                        //it could happen, that the article has only a start page
+                        startPage = element.getValue() + endPage;
+                        putIfNotNull(fields, "pages", startPage);
+                    } else if ("EndPage".equals(element.getName().getLocalPart())) {
+                        endPage = element.getValue();
+                        //but it should not happen, that a endpage appears without startpage
+                        fields.put("pages", fixPageRange(startPage + "-" + endPage));
+                    }
+                }
+            } else if (object instanceof ELocationID) {
+                ELocationID eLocationID = (ELocationID) object;
+                if ("doi".equals(eLocationID.getEIdType())) {
+                    fields.put("doi", eLocationID.getContent());
+                }
+                if ("pii".equals(eLocationID.getEIdType())) {
+                    fields.put("pii", eLocationID.getContent());
+                }
+            } else if (object instanceof Abstract) {
+                Abstract abs = (Abstract) object;
+                putIfNotNull(fields, "copyright", abs.getCopyrightInformation());
+                List<String> abstractText = new ArrayList<>();
+                for (AbstractText text : abs.getAbstractText()) {
+                    for (Serializable textContent : text.getContent()) {
+                        if (textContent instanceof String) {
+                            abstractText.add((String) textContent);
+                        }
+                    }
+                }
+                fields.put("abstract", join(abstractText, " "));
+            } else if (object instanceof AuthorList) {
+                AuthorList authors = (AuthorList) object;
+                handleAuthors(fields, authors);
+            }
+        }
+    }
+
+    private String extractYear(String medlineDate) {
+        //The year of the medlineDate should be the first 4 digits
+        return medlineDate.substring(0, 4);
+    }
+
+    private String removeSurroundingBrackets(String title) {
+        if (title.startsWith("[") && title.endsWith("]")) {
+            return title.replace("[", "").replace("]", "");
+        }
+        return title;
```

Comment:
There is also RemoveBracesFormatter which does a similar job (and is a bit more intelligent, I think). For example your code turns `[te[st]` into `test` and not `te[st`.